src/View.Uno.WinUI/View.Uno.WinUI.csproj

```diff
@@ -75,10 +75,115 @@
 		<_AndroidApiLevel>31</_AndroidApiLevel>
 	</PropertyGroup>
 
+	<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard2.0|AnyCPU'">
+	  <DefineConstants>$(DefineConstants);__WASM__;__NETSTANDARD__</DefineConstants>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard2.0|AnyCPU'">
+	  <DefineConstants>$(DefineConstants);__WASM__;__NETSTANDARD__</DefineConstants>
+	</PropertyGroup>
+
 	<ItemGroup>
 		<UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <PRIResource Remove="Converters\PrettyDateConverter\Resources\de\PrettyDateFormatterStrings.resw" />
```

Comment:
Could we reduce those `PRIResource Remove` to 1 line by using a globe pattern?

Reply:
And actually, the same comment applies for the `None Include` and the `EmbededResource`.

Reply:
Done.